tsconfig.json

```diff
@@ -7,5 +7,7 @@
   "compilerOptions": {
     // output to dist/ dir
     "outDir": "dist/",
+    "jsx": "react-jsx",
+    "importsNotUsedAsValues": "preserve"
```

Comment:
I don't think [this](https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues) is needed as all of the imports are still used even when the types are removed. Is there a reason you added this?